trunk/SUAVE/Methods/Noise/Fidelity_One/Noise_Tools/noise_geometric.py

```diff
@@ -150,4 +150,57 @@ def noise_geometric(noise_segment,analyses,config):
     noise_segment.theta = theta
     noise_segment.phi   = phi
 
-    return (dist,theta,phi)
\ No newline at end of file
+    return (dist,theta,phi)
+
+
+
+def geometric_propeller(noise_data):
+    """ SUAVE.Methods.Noise.Fidelity_One.Noise_Tools.geometric_propeller(noise_data):
+                Computes the geometric parameters for the noise tools: distance and emission angles for both polar and azimuthal angles.
+    
+                Inputs:
+                    noise_data	 - SUAVE type vehicle
+    
+                Outputs:
+                    dist                         - Distance vector from the aircraft position in relation to the microphone coordinates, [meters]
+                    theta                        - Polar angle emission vector relatively to the aircraft to the microphone coordinates, [rad]
+                    phi                          - Azimuthal angle emission vector relatively to the aircraft to the microphone coordinates, [rad]
+    
+                Assumptions:
+                    Propeller-driven aircraft has to comply only with the take-off noise limit."""    
+    
+    
+    # unpack
+    position_vector = noise_data.position
+    altitude        = noise_data.altitude
+    S_0             = noise_data.tofl * Units.ft
+    
+ #   s       = position_vector
+    n_steps = len(altitude)  #number of time steps (space discretization)    
+    
+    
+    #--------------------------------------------------------
+    #------------------ FLYOVER CALCULATION -----------------
+    #--------------------------------------------------------
+    
+    # Azimuthal angle is zero for flyover condition
+    phi=np.zeros(n_steps)    
+    theta = np.zeros(n_steps)  
+       
+    
+  #  S_0 = 1500 * Units.ft
```

Comment:
commented

Reply:
corrected - thanks for looking through these old files that were always regressed but never examined